pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -785,3 +792,24 @@ func (c *Cluster) getOSDLabels(osdID int, failureDomainValue string, portable bo
 		portableKey:         strconv.FormatBool(portable),
 	}
 }
+
+//GetVolumeGroupName returns the Volume group name from the given Logical Volume Path
+func GetVolumeGroupName(lvPath string) (string, error) {
+	if lvPath == "" || strings.Split(lvPath, "/")[2] == "" {
```

Comment:
After calling `strings.Split()`, need to check its length before calling `[2]`

Reply:
updated. Adding more strict conditions for lv path format.